src/agency.ts

```diff
@@ -17,7 +18,7 @@ export class Agency {
 
   async post<Result = any, Entity = any>(
     path: string,
-    entity: Entity,
+    entity?: Entity,
```

Comment:
```suggestion
    entity: Entity,
```

I think we need to keep this as a required value, as a post request should always contain a body.

Reply:
If we are not passing in any data then setting an empty `{}` value should work. eg. `post('/user', {})`